.gitignore

```diff
@@ -1,6 +1,7 @@
 /target
 Cargo.lock
 near-plugins-derive/tests/contracts/*/target
+near-plugins/tests/contracts/*/target
```

Comment:
This directory doesn't exist anymore in the repo, `near-plugins/tests` has been moved to `near-plugins-derive/tests`. Probably there are some stale files on your machine (fix `rm -r near-plugins/tests`).

Reply:
59245225e53f1deee0c14a5c67e69b9cb5d5346a